modules/nf-core/learnmsa/align/tests/main.nf.test

```diff
@@ -26,10 +27,33 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert path(process.out.alignment.get(0).get(1)).getText().contains(">sample1") },
-                { assert snapshot(process.out.versions).match("versions_structure") }
+                { assert snapshot(process.out.versions).match("versions") }
             )
         }
 
     }
 
-}
+    test("sarscov2 - fasta - compressed") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                             file(params.test_data['sarscov2']['genome']['informative_sites_fas'], checkIfExists: true)
+                           ]
+                input[1] = true
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.alignment.get(0).get(1)).getTextGzip().contains(">sample1") },
+                { assert snapshot(process.out.versions).match("versions1") }
```

Comment:
Ah nvm, for the snapshot name of the versions snapshot it's because since v 0.8.4 nf-test throws an error when overwriting a snapshot with the same name – compare [0].
This was causing the CI to fail despite the test passing locally (where I have nf-test v 0.8.3 installed).

[0] https://github.com/askimed/nf-test/commit/d2508b204cd3bd04242d566ccb5d6eaf48818400#diff-fd31905853152ac935c5509db52bb497033040fa8cb2523810950c3595ba450aR31-R37